### PR TITLE
Feature/guest picker mockup

### PR DESCRIPTION
**Pull request by lighthurst**
Hi @briansai Please review this PR. Thank you. https://trello.com/c/MxL1KuNx/17-mock-up-guests-and-date-input-fields

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -15,25 +14,34 @@ class App extends Component {
...
   }
   postBooking () {
 
   }
-  updateState (data) {
+  updateState(data) {
     const { avg_rating, total_ratings, max_guests, min_night_stay, cleaning_fee, addtl_guest_fee } = data[0][0];
-    const avgNightlyRate = data[1].reduce((acc, night) => acc = night.rate > acc ? night.rate : acc, 0);
+    const avgNightlyRate = data[1].reduce((acc, night) => {
+      acc = night.rate > acc ? night.rate : acc;
```
Comment (briansai) [inline client/src/components/App.jsx:40]: I like this ternary format.  I might have to try this out if it's ever applicable.

File: client/src/components/BookItForm.jsx [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
   </FormContainer>
 );
 
+BookItForm.propTypes = {
+  guestDropDownActive: PropTypes.bool.isRequired,
+  handleGuestDropDown: PropTypes.func.isRequired,
+  maxGuests: PropTypes.number.isRequired,
+};
+
 const FormContainer = styled.form`
```
Comment (briansai) [inline client/src/components/BookItForm.jsx:28]: Assuming there will be content in here and will be exported.

File: client/src/components/BookItFields.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+);
+
+BookItFields.propTypes = {
+  guestDropDownActive: PropTypes.bool.isRequired,
+  handleGuestDropDown: PropTypes.func.isRequired,
+  maxGuests: PropTypes.number.isRequired,
+};
+
+const DivContainer = styled.div`
+
```
Comment (briansai) [inline client/src/components/BookItFields.jsx:25]: Assuming there will be content in here and will be exported.

File: client/src/components/GuestPicker.jsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  handleGuestDropDown: PropTypes.func.isRequired,
+  maxGuests: PropTypes.number.isRequired,
+};
+
+const DivOuterContainer = styled.div`
+  margin-bottom: 16px;
+`;
+
+const LabelOuterContainer = styled.label`
+  display:block;
```
Comment (briansai) [inline client/src/components/GuestPicker.jsx]: add a space between colon and block

File: client/src/components/GuestPickerRowLabel.jsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  word-wrap: break-word;
+`;
+
+const DivSecondaryLabelContainer = DivLabelContainer.extend`
+  color: #484848;
+  font-size: 14px;
+  font-weight: normal;
+  letter-spacing: normal;
+  line-height: 18px;
+  padding-top: 4px
```
Comment (briansai) [inline client/src/components/GuestPickerRowLabel.jsx]: Semicolon after 4px

File: client/src/components/NightlyRate.jsx [hunk 6]
```diff
@@ -10,6 +11,10 @@ const NightlyRate = ({ avgNightlyRate }) => (
   </DivOuterContainer>
 );
 
+NightlyRate.propTypes = {
+  avgNightlyRate: PropTypes.number.isRequired,
+};
+
 const DivOuterContainer = styled.div`
 `;
```
Comment (briansai) [inline client/src/components/NightlyRate.jsx:19]: Assuming there will be content here and will be exported.

File: client/src/components/NightlyRateAndStars.jsx [hunk 7]
```diff
@@ -1,16 +1,23 @@
...
   </DivOuterContainer>
 );
 
+NightlyRateAndStars.propTypes = {
+  avgNightlyRate: PropTypes.number.isRequired,
+  stars: PropTypes.number.isRequired,
+  totRatings: PropTypes.number.isRequired,
+};
+
 const DivOuterContainer = styled.div`
```
Comment (briansai) [inline client/src/components/NightlyRateAndStars.jsx:22]: Assuming there will be content here and will be exported.

**Review by briansai — APPROVED** [review]
There are some minor changes needed, but definitely approved.  Great job Aaron!

File: client/src/components/App.jsx (hunk 1 above)
Comment (lighthurst) [reply client/src/components/App.jsx:40]: Thanks Brian!

**Pull request merged**